app/views/RoomView/index.js

```diff
@@ -58,7 +58,7 @@ class RoomView extends React.Component {
 	static navigationOptions = ({ navigation }) => {
 		const rid = navigation.getParam('rid');
 		const prid = navigation.getParam('prid');
-		const title = navigation.getParam('name');
+		const title = navigation.state.params.title || navigation.getParam('name');
```

Comment:
You not need to set two different params.
You can use only `navigation.getParam('name');`

Reply:
Thanks for the review Sir. I have made the changes.